download/releaseNotes/releaseNotes7.adoc

```diff
@@ -5,25 +5,6 @@
 :awestruct-release_notes_version: 7
 :awestruct-release_notes_version_qualifier: Final
 
-== New and noteworthy: Engine 7.44.0.Final
-
-=== Deprecated JavaScript support in configuration
-
-In this release, we have deprecated the rarely used ability to specify thread pool sizes using JavaScript expressions.
-We encourage everyone to check out our link:../upgradeRecipe/upgradeRecipe7.html[updated upgrade recipe] to adapt accordingly.
-
-Furthermore, users who wish to continue to use JavaScript configuration together with JDK 15 or later will have to add
-the following dependency to their runtime, on account of the OpenJDK project
-link:https://openjdk.java.net/jeps/372[removing their long since deprecated JavaScript support]:
-
-[source,xml,options="nowrap"]
-----
-<dependency>
-    <groupId>org.mozilla</groupId>
-    <artifactId>rhino</artifactId>
-</dependency>
-----
-
```

Comment:
I moved this to the upgrade recipe, because the "upgrade recipe" is the "here's the work you need to do" place (AKA the "bad news" place, although normally the work brings great benefits).
Also, this will affect less than 0.01% of the users, so it's not noteworthy enough to be in "new and noteworthy".